addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/xml/SystemControlXML.java

```diff
@@ -19,50 +19,55 @@
 import org.w3c.dom.Document;
 import org.w3c.dom.Node;
 
+import static org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLConstants.ON;
+import static org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLConstants.POWER_STANDBY;
+import static org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLUtils.getNode;
+import static org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLUtils.getNodeContentOrDefault;
+import static org.openhab.binding.yamahareceiver.internal.protocol.xml.XMLUtils.getNodeContentOrEmpty;
+
 /**
  * The system control protocol class is used to control basic non-zone functionality
  * of a Yamaha receiver with HTTP/xml.
  * No state will be saved in here, but in {@link SystemControlState} instead.
  *
  * @author David Gräff - Initial contribution
- * @author Tomasz Maruszak - refactoring
+ * @author Tomasz Maruszak - refactoring, HTR-xxxx Zone_2 compatibility
+ *
  */
 public class SystemControlXML implements SystemControl {
     private WeakReference<AbstractConnection> comReference;
     private SystemControlStateListener observer;
 
+    protected String setPowerCmd = "<System><Power_Control><Power>%s</Power></Power_Control></System>";
```

Comment:
This feels more like a constant, can it be one? Can it be private?

Reply:
It's not used as a constant. In specialized implementations of SystemControlXML for certain Yamaha models these commands are replaced with a custom command for what the particular AVR model requires.

Reply:
ok